app/javascript/src/components/Dashboard/Notes/NoteTable.jsx

```diff
@@ -1,5 +1,12 @@
 import React from "react";
-import { Checkbox } from "neetoui";
+import { Avatar, Badge, Button, Checkbox, Tooltip } from "neetoui";
+import format from "date-fns/format";
+
+const issueTypeBadgeColors = {
+  internal: "blue",
+  agile: "green",
+  bug: "red",
+};
```

Comment:
Please move this and any other constants to a separate constants file.

Reply:
Resolved in #9